scss/bitstyles/ui/sidebar.stories.mdx

```diff
@@ -81,73 +81,74 @@ Uses a fullwidth dropdown to fit with no overlap.
     {`
       <div class="u-flex u-height-100vh" x-data="{ sidebarOpen: false }">
```

Comment:
Can we build a sidebar component here, which accepts the content slots (that we want to provide different content to, so perhaps the logo can be hardcoded) as props? Then this story could be moved to a JS file, and imported into the mdx (as we transition to storybook 7, we’ll need to do this)